### PR TITLE
disabling human readable mode in grid

### PR DESCRIPTION
**Pull request by wheresrhys**

### CODE REVIEW

File: grid/README.md [hunk 1]
```diff
@@ -23,4 +21,4 @@ e.g.  `@include nUiGridOffset(M, 3, pull)`
 
 ## Human readable selectors
 
-Pending a pull request to o-grid, these will be disabled, so use the numeric selectors i.e. `data-o-grid-colspan="6"` not `data-o-grid-colspan="half"`. The one exception is the `hide` keyword
+Pending a pull request to o-grid, these will be disabled, so use the numeric selectors i.e. `data-o-grid-colspan="6"` not `data-o-grid-colspan="half"`. Notably, use `0` instead of `hide`.
```
Comment (i-like-robots) [inline grid/README.md]: I think this will break things =[

Comment (wheresrhys) [inline grid/README.md]: Why? `0` doesn't actually work, but should (just an alias for `hide`)

Needs a PR in o-grid to uncouple from the other human readable selectors though

Comment (wheresrhys) [inline grid/README.md]: Either way, not planning on merging til that's fixed upstream

Comment (i-like-robots) [inline grid/README.md]: I mean I believe we use `hide` a fair bit in our templates

**Pull request closed without merging**